### PR TITLE
Update use-network-upload-to-import-pst-files.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the navigation as it has changed

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4295

File: microsoft-365/compliance/use-network-upload-to-import-pst-files.md [hunk 1]
```diff
@@ -177,9 +177,7 @@ To install the Azure Storage Explorer and connect to your Azure Storage area:
   
 1. Download and install the [Microsoft Azure Storage Explorer tool](https://go.microsoft.com/fwlink/p/?LinkId=544842).
     
-2. Start the Microsoft Azure Storage Explorer, right-click **Storage Accounts** in the left pane, and then click **Connect to Azure Storage**.
-    
-    ![Right-click Storage Accounts and then click Connect to Azure Storage](../media/75b80cc3-c336-4f96-ad32-54ac9b96a7af.png)
+2. Start the Microsoft Azure Storage Explorer, Choose **Blob Container**
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/use-network-upload-to-import-pst-files.md]:
```suggestion
2. Start the Microsoft Azure Storage Explorer and select **Blob Container**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**